learning-another-language/variables.md

```diff
@@ -68,6 +68,41 @@ var jingsFavColor = 'green';
 ```
 <!-- prettier-ignore-end -->
 
+### Scope of var vs let & const
+
+When creating a variable using `let` and `const` will scope the variable to the block it is in.  In the below example `i` and `name` are defined inside of the `for-of` loop and are scoped to the loop so are not accessable outside of the loop.  This applies to all blocks of code in JavaScript, except conditionals (if statements).  
+
+```javaScript
+const nums = [14, 15, 16, 18, 19, 20];
+
+for (let i of nums) {
+    console.log(`i is ${i}`);
+    const name = 'peter';
+    console.log(`name is ${name}`);
+}
+
+console.log(i); // ReferenceError: i is not defined
+console.log(name); // ReferenceError: name is not defined
+
+```
+
+On the other hand, when using `var` the variable is scoped to the function it is defined in.
```

Comment:
Strengthen the contrast a bit. Also, this example doesn't show the code being in a function, which might be confusing without acknowledging that.

```suggestion
On the other hand, when using `var`, the variable is instead scoped to the _function_ that contains it, or if not in a function, to the global scope.
```